curriculum/challenges/_meta/learn-css-colors-by-building-a-set-of-colored-markers/meta.json

```
@@ -0,0 +1,390 @@
+{
+  "name": "Learn CSS Colors by Building a Set of Colored Markers",
+  "isUpcomingChange": false,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
+  "dashedName": "learn-css-colors-by-building-a-set-of-colored-markers",
+  "order": 2,
+  "time": "5 hours",
+  "template": "",
+  "required": [],
+  "superBlock": "responsive-web-design",
```

```suggestion
  "superBlock": "2022/responsive-web-design",
```
Since Oliver's pathing PR has landed.

---

Haha, just rebased this